GameServerLib/GameObjects/Other/Target.cs

```diff
@@ -47,5 +47,11 @@ public Vector2 GetPosition()
         {
             return new Vector2(X, Y);
         }
+
+        public bool WithinRange(Vector2 from, Vector2 to, float range)
```

Comment:
Really should be static and be putted in some other helper class.

Or to use the ITarget position somehow.

Reply:
Would this be better in AttackableUnit?

Reply:
try to do that as extended function like:
`to.WithinRange(from,range)`

Reply:
For another PR.